src/lsm/segmented_array.zig

```diff
@@ -1078,7 +1079,8 @@ fn FuzzContextType(
         removes: u64 = 0,
 
         fn init(allocator: std.mem.Allocator, random: std.rand.Random) !Self {
-            var pool = try TestPool.init(allocator, TestArray.node_count_max);
```

Comment:
Same here, we are in-place initing pool, so the entire FuzzContext has to be in-place